core/parachain/pvf/workers.cpp

```diff
@@ -119,7 +119,6 @@ namespace kagome::parachain {
         scheduler_{std::move(scheduler)},
         exe_{exePath()},
         max_{app_config.pvfMaxWorkers()},
-        timeout_{app_config.pvfSubprocessDeadline()},
```

Comment:
If we are not using pvf subprocess deadline anymore it should be removed from CLI as well (app_configuration_impl)

Reply:
Done